v0.2.0 fails to build with libc++

Regressed by 766d47030890. Can be fixed by adding `<algorithm>`:
```c++
src/core/hyprlock.cpp:872:17: error: no member named 'count_if' in namespace 'std'
  872 |     return std::count_if(m_sPasswordState.passBuffer.begin(), m_sPasswordState.passBuffer.end(), [](char c) { return (c & 0xc0) != 0x80; });
      |            ~~~~~^
```

Regressed by 149b6737c263. Tested with libc++ 18.1.1. No clue how to fix:
```c++
src/renderer/widgets/PasswordInputField.cpp:419:116: error: invalid operands to binary expression ('time_point' (aka 'time_point<std::chrono::steady_clock, duration<long long, ratio<1LL, 1000000000LL>>>') and 'time_point' (aka 'time_point<system_clock>'))
  419 |         std::clamp(std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::high_resolution_clock::now() - TIMER).count() / (double)outerColor.transitionMs, 0.02, 0.5);
      |                                                                          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ ^ ~~~~~
/usr/include/c++/v1/__chrono/day.h:75:44: note: candidate function not viable: no known conversion from 'time_point' (aka 'time_point<std::chrono::steady_clock, duration<long long, ratio<1LL, 1000000000LL>>>') to 'const day' for 1st argument
   75 | _LIBCPP_HIDE_FROM_ABI inline constexpr day operator-(const day& __lhs, const days& __rhs) noexcept {
      |                                            ^         ~~~~~~~~~~~~~~~~
/usr/include/c++/v1/__chrono/day.h:79:45: note: candidate function not viable: no known conversion from 'time_point' (aka 'time_point<std::chrono::steady_clock, duration<long long, ratio<1LL, 1000000000LL>>>') to 'const day' for 1st argument
   79 | _LIBCPP_HIDE_FROM_ABI inline constexpr days operator-(const day& __lhs, const day& __rhs) noexcept {
      |                                             ^         ~~~~~~~~~~~~~~~~
/usr/include/c++/v1/__chrono/month.h:77:46: note: candidate function not viable: no known conversion from 'time_point' (aka 'time_point<std::chrono::steady_clock, duration<long long, ratio<1LL, 1000000000LL>>>') to 'const month' for 1st argument
   77 | _LIBCPP_HIDE_FROM_ABI inline constexpr month operator-(const month& __lhs, const months& __rhs) noexcept {
      |                                              ^         ~~~~~~~~~~~~~~~~~~
/usr/include/c++/v1/__chrono/month.h:81:47: note: candidate function not viable: no known conversion from 'time_point' (aka 'time_point<std::chrono::steady_clock, duration<long long, ratio<1LL, 1000000000LL>>>') to 'const month' for 1st argument
   81 | _LIBCPP_HIDE_FROM_ABI inline constexpr months operator-(const month& __lhs, const month& __rhs) noexcept {
      |                                               ^         ~~~~~~~~~~~~~~~~~~
/usr/include/c++/v1/__chrono/weekday.h:108:48: note: candidate function not viable: no known conversion from 'time_point' (aka 'time_point<std::chrono::steady_clock, duration<long long, ratio<1LL, 1000000000LL>>>') to 'const weekday' for 1st argument
  108 | _LIBCPP_HIDE_FROM_ABI inline constexpr weekday operator-(const weekday& __lhs, const days& __rhs) noexcept {
      |                                                ^         ~~~~~~~~~~~~~~~~~~~~
/usr/include/c++/v1/__chrono/weekday.h:112:45: note: candidate function not viable: no known conversion from 'time_point' (aka 'time_point<std::chrono::steady_clock, duration<long long, ratio<1LL, 1000000000LL>>>') to 'const weekday' for 1st argument
  112 | _LIBCPP_HIDE_FROM_ABI inline constexpr days operator-(const weekday& __lhs, const weekday& __rhs) noexcept {
      |                                             ^         ~~~~~~~~~~~~~~~~~~~~
/usr/include/c++/v1/__chrono/year.h:87:45: note: candidate function not viable: no known conversion from 'time_point' (aka 'time_point<std::chrono::steady_clock, duration<long long, ratio<1LL, 1000000000LL>>>') to 'const year' for 1st argument
   87 | _LIBCPP_HIDE_FROM_ABI inline constexpr year operator-(const year& __lhs, const years& __rhs) noexcept {
      |                                             ^         ~~~~~~~~~~~~~~~~~
/usr/include/c++/v1/__chrono/year.h:91:46: note: candidate function not viable: no known conversion from 'time_point' (aka 'time_point<std::chrono::steady_clock, duration<long long, ratio<1LL, 1000000000LL>>>') to 'const year' for 1st argument
   91 | _LIBCPP_HIDE_FROM_ABI inline constexpr years operator-(const year& __lhs, const year& __rhs) noexcept {
      |                                              ^         ~~~~~~~~~~~~~~~~~
/usr/include/c++/v1/__chrono/year_month.h:84:47: note: candidate function not viable: no known conversion from 'time_point' (aka 'time_point<std::chrono::steady_clock, duration<long long, ratio<1LL, 1000000000LL>>>') to 'const year_month' for 1st argument
   84 | _LIBCPP_HIDE_FROM_ABI inline constexpr months operator-(const year_month& __lhs, const year_month& __rhs) noexcept {
      |                                               ^         ~~~~~~~~~~~~~~~~~~~~~~~
/usr/include/c++/v1/__chrono/year_month.h:89:51: note: candidate function not viable: no known conversion from 'time_point' (aka 'time_point<std::chrono::steady_clock, duration<long long, ratio<1LL, 1000000000LL>>>') to 'const year_month' for 1st argument
   89 | _LIBCPP_HIDE_FROM_ABI inline constexpr year_month operator-(const year_month& __lhs, const months& __rhs) noexcept {
      |                                                   ^         ~~~~~~~~~~~~~~~~~~~~~~~
/usr/include/c++/v1/__chrono/year_month.h:93:51: note: candidate function not viable: no known conversion from 'time_point' (aka 'time_point<std::chrono::steady_clock, duration<long long, ratio<1LL, 1000000000LL>>>') to 'const year_month' for 1st argument
   93 | _LIBCPP_HIDE_FROM_ABI inline constexpr year_month operator-(const year_month& __lhs, const years& __rhs) noexcept {
      |                                                   ^         ~~~~~~~~~~~~~~~~~~~~~~~
/usr/include/c++/v1/__chrono/year_month_day.h:155:1: note: candidate function not viable: no known conversion from 'time_point' (aka 'time_point<std::chrono::steady_clock, duration<long long, ratio<1LL, 1000000000LL>>>') to 'const year_month_day' for 1st argument
  155 | operator-(const year_month_day& __lhs, const months& __rhs) noexcept {
      | ^         ~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/include/c++/v1/__chrono/year_month_day.h:170:1: note: candidate function not viable: no known conversion from 'time_point' (aka 'time_point<std::chrono::steady_clock, duration<long long, ratio<1LL, 1000000000LL>>>') to 'const year_month_day' for 1st argument
  170 | operator-(const year_month_day& __lhs, const years& __rhs) noexcept {
      | ^         ~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/include/c++/v1/__chrono/year_month_day.h:304:1: note: candidate function not viable: no known conversion from 'time_point' (aka 'time_point<std::chrono::steady_clock, duration<long long, ratio<1LL, 1000000000LL>>>') to 'const year_month_day_last' for 1st argument
  304 | operator-(const year_month_day_last& __lhs, const months& __rhs) noexcept {
      | ^         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/include/c++/v1/__chrono/year_month_day.h:319:1: note: candidate function not viable: no known conversion from 'time_point' (aka 'time_point<std::chrono::steady_clock, duration<long long, ratio<1LL, 1000000000LL>>>') to 'const year_month_day_last' for 1st argument
  319 | operator-(const year_month_day_last& __lhs, const years& __rhs) noexcept {
      | ^         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/include/c++/v1/__chrono/year_month_weekday.h:131:1: note: candidate function not viable: no known conversion from 'time_point' (aka 'time_point<std::chrono::steady_clock, duration<long long, ratio<1LL, 1000000000LL>>>') to 'const year_month_weekday' for 1st argument
  131 | operator-(const year_month_weekday& __lhs, const months& __rhs) noexcept {
      | ^         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/include/c++/v1/__chrono/year_month_weekday.h:146:1: note: candidate function not viable: no known conversion from 'time_point' (aka 'time_point<std::chrono::steady_clock, duration<long long, ratio<1LL, 1000000000LL>>>') to 'const year_month_weekday' for 1st argument
  146 | operator-(const year_month_weekday& __lhs, const years& __rhs) noexcept {
      | ^         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/include/c++/v1/__chrono/year_month_weekday.h:241:1: note: candidate function not viable: no known conversion from 'time_point' (aka 'time_point<std::chrono::steady_clock, duration<long long, ratio<1LL, 1000000000LL>>>') to 'const year_month_weekday_last' for 1st argument
  241 | operator-(const year_month_weekday_last& __lhs, const months& __rhs) noexcept {
      | ^         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/include/c++/v1/__chrono/year_month_weekday.h:256:1: note: candidate function not viable: no known conversion from 'time_point' (aka 'time_point<std::chrono::steady_clock, duration<long long, ratio<1LL, 1000000000LL>>>') to 'const year_month_weekday_last' for 1st argument
  256 | operator-(const year_month_weekday_last& __lhs, const years& __rhs) noexcept {
      | ^         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/include/c++/v1/__chrono/time_point.h:213:1: note: candidate template ignored: deduced conflicting types for parameter '_Clock' ('std::chrono::steady_clock' vs. 'std::chrono::system_clock')
  213 | operator-(const time_point<_Clock, _Duration1>& __lhs, const time_point<_Clock, _Duration2>& __rhs) {
      | ^
/usr/include/c++/v1/__iterator/wrap_iter.h:182:5: note: candidate template ignored: could not match '__wrap_iter' against 'time_point'
  182 |     operator-(const __wrap_iter<_Iter1>& __x,
      |     ^
/usr/include/c++/v1/__iterator/reverse_iterator.h:288:1: note: candidate template ignored: could not match 'reverse_iterator' against 'time_point'
  288 | operator-(const reverse_iterator<_Iter1>& __x, const reverse_iterator<_Iter2>& __y)
      | ^
/usr/include/c++/v1/__ios/fpos.h:60:40: note: candidate template ignored: could not match 'fpos' against 'time_point'
   60 | inline _LIBCPP_HIDE_FROM_ABI streamoff operator-(const fpos<_StateT>& __x, const fpos<_StateT>& __y) {
      |                                        ^
/usr/include/c++/v1/__chrono/duration.h:405:5: note: candidate template ignored: could not match 'duration' against 'time_point'
  405 |     operator-(const duration<_Rep1, _Period1>& __lhs, const duration<_Rep2, _Period2>& __rhs) {
      |     ^
/usr/include/c++/v1/__chrono/time_point.h:204:5: note: candidate template ignored: could not match 'duration' against 'time_point'
  204 |     operator-(const time_point<_Clock, _Duration1>& __lhs, const duration<_Rep2, _Period2>& __rhs) {
      |     ^
/usr/include/c++/v1/__iterator/move_iterator.h:304:1: note: candidate template ignored: could not match 'move_iterator' against 'time_point'
  304 | operator-(const move_iterator<_Iter1>& __x, const move_iterator<_Iter2>& __y) -> decltype(__x.base() - __y.base()) {
      | ^
```

--- a/src/core/hyprlock.cpp
+++ b/src/core/hyprlock.cpp
@@ -16,6 +16,7 @@
 #include <xf86drm.h>
 #include <filesystem>
 #include <fstream>
+#include <algorithm>
 
 CHyprlock::CHyprlock(const std::string& wlDisplay, const bool immediate) {
     m_sWaylandState.display = wl_display_connect(wlDisplay.empty() ? nullptr : wlDisplay.c_str());

--- a/src/renderer/widgets/PasswordInputField.cpp
+++ b/src/renderer/widgets/PasswordInputField.cpp
@@ -416,7 +416,7 @@ void CPasswordInputField::updateOuter() {
 
     const auto MULTI = outerColor.transitionMs == 0 ?
         1.0 :
-        std::clamp(std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::high_resolution_clock::now() - TIMER).count() / (double)outerColor.transitionMs, 0.02, 0.5);
+        std::clamp(std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::system_clock::now() - TIMER).count() / (double)outerColor.transitionMs, 0.02, 0.5);
     const auto DELTA = TARGET - SOURCE;
 
     if (outerColor.main.r != TARGET.r) {

